qutebrowser/utils/log.py

```diff
@@ -149,6 +149,7 @@ class CriticalQtWarning(Exception):
 
 def init_log(args):
     """Init loggers based on the argparse namespace passed."""
+    global console
```

Comment:
You already set the global `console_handler` in `_init_handlers`, which is the same as `console` - why not just use that everywhere?